Week1/subway/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+  <head></head>
+  <style>
+	#map {
+	   height: 100%;
+	 }
+  </style>
+  <body>
+	<div id="map"></div>
+  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script>
+	let map;
+	function getLatLng(data){
+		//entrances
+		const entrances = data[0];
+		let entlocations = [];
+		for(let e of entrances){
+			entlocations.push(e["the_geom"]);
+		}
+		//stations
+		const stations = data[1];
+		let stalocations = [];
+		for(let s of stations){
+			stalocations.push(s["the_geom"]);
+		} // I know there is a better way to do this, but I forgot !
+		return [entlocations, stalocations];
+	}
+	function getData(callback){
+	  $.ajax({
+			url: "http://localhost:8000/subways.json",
```

Comment:
```js
  url: "subways.json",
```

Unfortunately `localhost` isn't accessible on the web as it's your personal computer. The IP address for that is `127.0.0.1` which is universal to everyone. So you are saying, go to your personal computer and look for the `subways.json` file. That doesn't exist, so instead we need to remove that from the URL. Now it states, whatever server we are on, look for the `subways.json` file.